snapcraft.yaml

```diff
@@ -16,10 +16,11 @@ parts:
     source: https://github.com/juju/juju.git
```

Comment:
Same as below. Perhaps it should be the default for this snapcraft.yaml checked into the repo.